docs/guide/content/installation.md

```diff
@@ -177,3 +177,66 @@ your server names.
 1. Check the NodeJS modules are deployed `node_modules`
 1. Launch the OnDemand Loop application by visiting the URL: `https://<your-server>/pun/sys/loop` in a browser. The application should load after clicking **Initialize** once.
 1. The OnDemand Loop homepage should display with a welcome message.
+
+### Configuring Navigation Menu Appearance
+
+OnDemand Loop's appearance in the Open OnDemand navigation menu is controlled by the `manifest.yml` file located in the application directory. This configuration determines how the application appears to users in the OOD dashboard.
+
+#### Default Configuration
+
+The default `manifest.yml` contains:
+
+```yaml
+---
+name: OnDemand Loop
+category: Files
+description: |-
+  Transfer files from and to remote repositories like Dataverse.
+icon: fa://copy
+new_window: false
+```
+
+#### Customizing Navigation Placement
+
+You can customize how OnDemand Loop appears in the navigation menu by modifying the `manifest.yml` file:
+
+**Navigation Fields:**
+- **`name`** - Display name in navigation menus and icons
+- **`category`** - Primary menu category (creates dropdown menus)  
+- **`subcategory`** - Submenu grouping within categories (optional)
+- **`description`** - Tooltip and description text shown to users
+- **`icon`** - Icon displayed in menus (FontAwesome format)
+- **`new_window`** - Whether app opens in new window (`true`/`false`)
```

Comment:
I think your intention here is to list the fields in bullet points having each field in a new line. However, this section is being rendered as one line and I think it is better to fix this to put each one in one line

Reply:
fixed